webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/OrderedScrambles.java

```diff
@@ -32,6 +32,12 @@ public static void generateOrderedScrambles(String globalTitle, Date generationD
         }
         
         azzert(wcifHelper.getAllScrambleRequests() != null, "There should be scramble requests.");
+        
+        // allActivities hold activities without repetition.
+        // A ScrambleRequest is added to AllScramblesOrdered if its activity is not on allActivities.
+        // This is needed because the same activity may happen on different rooms.
```

Comment:
Why would an activity happening in different rooms not end up in `allActivities`?